powerauth-fido2/src/main/java/com/wultra/powerauth/fido2/rest/model/entity/Fido2Authenticators.java

```diff
@@ -0,0 +1,259 @@
+/*
+ * PowerAuth Server and related software components
+ * Copyright (C) 2023 Wultra s.r.o.
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU Affero General Public License as published
+ * by the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU Affero General Public License for more details.
+ *
+ * You should have received a copy of the GNU Affero General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ */
+
+package com.wultra.powerauth.fido2.rest.model.entity;
+
+import com.wultra.security.powerauth.client.model.enumeration.SignatureType;
+import lombok.Data;
+
+import java.nio.ByteBuffer;
+import java.util.*;
+
+/**
+ * Class containing map of all known FIDO2 AAGUID authenticator identifiers.
+ *
+ * @author Petr Dvorak, petr@wultra.com
+ */
+public final class Fido2Authenticators {
+
+    /**
+     * Model class representing an authenticator model. It associates the AAGUID value to a descriptive name
+     * and expected authentication factors available with a given authenticator. Most authenticators are set to
+     * provide only the possession factor on approval. In case the authenticator has a biometric sensor, it will be
+     * represented
+     */
+    @Data
+    public static class Model {
+
+        private final UUID aaguid;
+        private final String description;
+        private SignatureType signatureType;
```

Comment:
🤔 May it also be `final`? Could we rather use `record` instead of Lombok `Data`?

Reply:
Record seems fine.